packages/react-shortcuts/src/keys.ts

```diff
@@ -0,0 +1,445 @@
+type Key =
```

Comment:
Looks like we should close the other PR that has this stuff?

Reply:
yes, I'll merge that PR in just before this one, just don't wanna do two separate releases :sweat_smile: